recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2.2.2" %}
+{% set version = "2.2.3" %}
```

Comment:
Is this correct? 

https://pypi.org/project/numpy/#history

![image](https://github.com/user-attachments/assets/9a0400ce-42c8-47aa-a3c9-246cf8d982dd)

Reply:
It is already tagged: https://github.com/numpy/numpy/releases/tag/v2.2.3. Just not on PyPI yet. The bot looks at GitHub Releases I think, so it should be correct.